coral/src/app/features/approvals/acls/components/AclApprovalsTable.test.tsx

```diff
@@ -0,0 +1,263 @@
+import AclApprovalsTable from "src/app/features/approvals/acls/components/AclApprovalsTable";
+import { mockIntersectionObserver } from "src/services/test-utils/mock-intersection-observer";
+
+const aclRequests: AclRequest[] = [
+  {
+    remarks: undefined,
+    consumergroup: "-na-",
+    acl_ip: undefined,
+    acl_ssl: ["mbasani", "maulbach"],
+    aclPatternType: "LITERAL",
+    transactionalId: undefined,
+    req_no: 1014,
+    topicname: "aivtopic1",
+    environment: "1",
+    teamname: "Ospo",
+    aclIpPrincipleType: "PRINCIPAL",
+    environmentName: "DEV",
+    teamId: 1003,
+    requestingteam: 1003,
+    requestingTeamName: "Ospo",
+    appname: "App",
+    username: "amathieu",
+    requesttime: "2023-01-06T14:50:37.912+00:00",
+    requesttimestring: "06-Jan-2023 14:50:37",
+    requestStatus: "CREATED",
+    approver: undefined,
+    approvingtime: undefined,
+    aclType: "CONSUMER",
+    aclResourceType: undefined,
+    currentPage: "1",
+    otherParams: undefined,
+    totalNoPages: "2",
+    allPageNos: ["1", ">", ">>"],
+    approvingTeamDetails:
+      "Team : Ospo, Users : muralibasani,josepprat,samulisuortti,mirjamaulbach,smustafa,aindriul,",
+  },
+  {
+    remarks: "hello",
+    consumergroup: undefined,
+    acl_ip: ["3.3.3.32", "3.3.3.33"],
+    acl_ssl: ["User:*"],
+    aclPatternType: "PREFIXED",
+    transactionalId: undefined,
+    req_no: 1015,
+    topicname: "newaudittopic",
+    environment: "2",
+    teamname: "Ospo",
+    aclType: "PRODUCER",
+    aclIpPrincipleType: "IP_ADDRESS",
+    environmentName: "TST",
+    teamId: 1003,
+    requestingteam: 1003,
+    appname: "App",
+    username: "amathieu",
+    requesttime: "2023-01-10T13:19:10.757+00:00",
+    requesttimestring: "10-Jan-2023 13:19:10",
+    requestStatus: "APPROVED",
+    approver: undefined,
+    approvingtime: undefined,
+    aclResourceType: undefined,
+    currentPage: "1",
+    otherParams: undefined,
+    totalNoPages: "2",
+    allPageNos: ["1", ">", ">>"],
+    approvingTeamDetails:
+      "Team : Ospo, Users : muralibasani,josepprat,samulisuortti,mirjamaulbach,smustafa,aindriul,",
+  },
+];
+
+describe("AclApprovalsTable", () => {
+  let rows;
+  let header: HTMLElement;
+  let firstRow: HTMLElement;
+  let secondRow: HTMLElement;
+  let onApprove: jest.Mock;
+  let onDecline: jest.Mock;
+  let onDetails: jest.Mock;
+
+  beforeEach(() => {
+    mockIntersectionObserver();
+    onApprove = jest.fn();
+    onDecline = jest.fn();
+    onDetails = jest.fn();
+    render(
+      <AclApprovalsTable
+        aclRequests={aclRequests}
+        activePage={1}
+        totalPages={10}
+        onApprove={onApprove}
+        onDecline={onDecline}
+        onDetails={onDetails}
+      />
+    );
+    rows = screen.getAllByRole("row");
+    header = rows[0];
+    firstRow = rows[1];
+    secondRow = rows[2];
+  });
+
+  afterEach(cleanup);
+
+  it("describes the table content and pagination for screen readers", () => {
```

Comment:
The description implies the test confirms the table & pagination with accessible information, but it does not check the table :D 

I would split that up into two blocks: 
- table content
-> check that `getByRole("table")` is visible 

- pagination 
-> getByRole("navigation", { name: "label text here" }) is visible

Reply:
Yes, it could be more specific check that the matching element is a table. This test is just a check that the table has aria label with the pagination information.